flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/expressions/RexNodeConverter.java

```diff
@@ -79,25 +112,196 @@
 	private final RelBuilder relBuilder;
```

Comment:
> TODO actually we should use {@link ResolvedExpressionVisitor} here as it is the output of the API

You can comment clear, why it keep `ExpressionVisitor` instead of `ResolvedExpressionVisitor`.

Reply:
Sure. Will update to use ResolvedExpressionVisitor  after introduce Expression resolve in AggCodeGen later. which may be done in another pr.